valhalla/tyr/util.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <string>
+
+namespace valhalla {
+namespace tyr {
+
+// Returns the 3-char equivalent of the 2-char country code (iso_3166_1_alpha2) or an empty string
+// if the 2-char code is unknown
+std::string get_iso_3166_1_alpha3(const std::string& iso_3166_1_alpha2);
```

Comment:
this seems like an unfortunate expansion of the public api, can we just throw this into the compilation unit where it is used? or if we want to use it in more than one place can we make it a private header (ie move it to `src/tyr`)?

Reply:
i see you use it in triple g builder which makes me think th is should just go into like baldr/graphconstants or something

Reply:
yeah its needed in 2 places thor::triplegbuilder and tyr when serializing the response. I looked at midgard initially but it seemed specific to logging (?). Is putting it within baldr a good option?